### ISSUE TITLE
Addition in logs: Add shared node info

### ISSUE THREAD

**Issue by rioda-org** [idena-network/idena-web#420]
It would be usefull for troubleshooting if Web App logs would include shared node url that app is connected to. Also if the url changes, that would also be usefull.

**Comment by hlolve** [comment]
Good point, and also include in client logs since lottery start, client log currently start at 15:00 UTC (short session)
